apps/vc-api/test/vc-api/exchanges/resident-card/resident-card.e2e-suite.ts

```diff
@@ -56,9 +56,14 @@ export const residentCardExchangeSuite = () => {
     const didAuthVp = didAuthResponse.body;
     expect(didAuthVp).toBeDefined();
 
-    // Continue exchange by submitting presention
+    // As holder, continue exchange by submitting did auth presention
     await walletClient.continueExchange(issuanceExchangeContinuationEndpoint, didAuthVp, true);
 
+    // As the issuer, get the transaction
+    const urlComponents = issuanceExchangeContinuationEndpoint.split('/');
+    const transactionId = urlComponents[urlComponents.length - 1];
+    const transaction = await walletClient.getExchangeTransaction(exchange.getExchangeId(), transactionId);
+
     // TODO: have the issuer get the review and approve. For now, just issue directly
     const issueResult = await exchange.issueCredential(didAuthVp, walletClient);
```

Comment:
Maybe we can merge this two promises into Promise.all, to improve performance?

Reply:
The second promise actually depends on the first promise, it's just not shown yet. I will implement that next.
Good suggestion though.

Reply:
Ohh okay, fair enough